beads_db/src/main/resources/db_scripts/V1_1__productgroup.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE `productgroup` (
+  `ID` int(11) NOT NULL AUTO_INCREMENT,
+  `name` varchar(100) NOT NULL,
+  `PARENT_ID` int(11) DEFAULT NULL,
```

Comment:
named convention rule : 
capital letter to keywords like `NOT NULL AUTO_INCREMENT,  DEFAULT NULL, FROM, SELECT, WHERE ...`
columns, tables, bind variables lower case

Reply:
you can use `reformat code` option from idea and see the result